src/SDKs/IotCentral/Management.IotCentral/generate.ps1

```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "iotcentral/resource-manage" -AutoRestVersion "latest" -SdkRootDirectory $PSScriptRoot
```

Comment:
Please fix the `ResourceProvider` here, should be `iotcentral/resource-manager`

Reply:
Good catch, I've updated this and regenerated everything